components/manager/scripts/configure_manager.py

```diff
@@ -63,11 +63,34 @@ def _create_certs():
     utils.generate_internal_ssl_cert(ips=cert_ips, name=internal_rest_host)
 
 
+def create_cloudify_user():
```

Comment:
Go ahead and close this one https://cloudifysource.atlassian.net/browse/CFY-7209 :)